### PR TITLE
Update license notice to include VS Tools license

### PR DESCRIPTION
**Pull request by mthalman**

### CODE REVIEW

**Review by MichaelSimons — APPROVED** [review]

**Comment by richlander** [comment]
Given that we're now referencing two licenses, can you use a bulleted list? It makes sense to pull that legal notice into the same list.

**Review by richlander — APPROVED** [review]

File: README.aspnet.md [hunk 1]
```diff
@@ -117,5 +117,5 @@ See the [.NET Framework Lifecycle FAQ](https://support.microsoft.com/help/17455/
 # License
 
 * Legal Notice: [Container License Information](https://aka.ms/mcr/osslegalnotice)
```
Comment (richlander) [inline README.aspnet.md:119]: How about this instead:

* The [MICROSOFT SOFTWARE SUPPLEMENTAL LICENSE FOR WINDOWS CONTAINER BASE IMAGE](https://hub.docker.com/_/microsoft-windows-servercore/) applies to .NET Framework container images.
* The [Visual Studio Tools license](https://visualstudio.microsoft.com/license-terms/mlt031519/) applies to [.NET Framework SDK images](https://hub.docker.com/_/microsoft-dotnet-framework-sdk/).
* [General legal notice for Microsoft container images](https://aka.ms/mcr/osslegalnotice)

Comment (richlander) [reply README.aspnet.md:119]: We could then do the same thing for the ~/dotnet README (minus the VS one).

File: eng/readme-templates/README.md [hunk 2]
```diff
@@ -88,6 +88,6 @@ See the [.NET Framework Lifecycle FAQ](https://support.microsoft.com/help/17455/
 
 # License
 
-* Legal Notice: [Container License Information](https://aka.ms/mcr/osslegalnotice)
-* The .NET Framework images use the same license as the [Windows Server Core base image](https://hub.docker.com/_/microsoft-windows-servercore/).
-* The [Visual Studio Tools license](https://visualstudio.microsoft.com/license-terms/mlt031519/) also applies to the [SDK images](https://hub.docker.com/_/microsoft-dotnet-framework-sdk/).
+* The [MICROSOFT SOFTWARE SUPPLEMENTAL LICENSE FOR WINDOWS CONTAINER BASE IMAGE](https://hub.docker.com/_/microsoft-windows-servercore/) applies to [].NET Framework container images](https://hub.docker.com/_/microsoft-dotnet-framework/).
```
Comment (MichaelSimons) [inline eng/readme-templates/README.md:91]:
```suggestion
* The [MICROSOFT SOFTWARE SUPPLEMENTAL LICENSE FOR WINDOWS CONTAINER BASE IMAGE](https://hub.docker.com/_/microsoft-windows-servercore/) applies to [.NET Framework container images](https://hub.docker.com/_/microsoft-dotnet-framework/).
```

Comment (MichaelSimons) [reply eng/readme-templates/README.md:91]: Thoughts on saying `applies to all` to help provide clarification with the second bullet.

Comment (MichaelSimons) [reply eng/readme-templates/README.md]: I'm sorry Matt, my last comment was clear as mud!  I intended the `applied to all` to be added to the first bullet.

File: eng/readme-templates/README.md [hunk 3]
```diff
@@ -88,6 +88,6 @@ See the [.NET Framework Lifecycle FAQ](https://support.microsoft.com/help/17455/
 
 # License
 
-* Legal Notice: [Container License Information](https://aka.ms/mcr/osslegalnotice)
-
-The .NET Framework images use the same license as the [Windows Server Core base image](https://hub.docker.com/_/microsoft-windows-servercore/).
+* The [MICROSOFT SOFTWARE SUPPLEMENTAL LICENSE FOR WINDOWS CONTAINER BASE IMAGE](https://hub.docker.com/_/microsoft-windows-servercore/) applies to all [.NET Framework container images](https://hub.docker.com/_/microsoft-dotnet-framework/).
```
Comment (MichaelSimons) [inline eng/readme-templates/README.md:91]: I'm going to channel @dleeapho with this comment.  It is not ideal that the first two bullets are complete sentences while the third is not.  What about something like the following?  This format feels more consistent with the other list sections within the readme.

```
* Legal Notice: [Container License Information](https://aka.ms/mcr/osslegalnotice)
* [MICROSOFT SOFTWARE SUPPLEMENTAL LICENSE FOR WINDOWS CONTAINER BASE IMAGE](https://hub.docker.com/_/microsoft-windows-servercore/): applies to all [.NET Framework container images](https://hub.docker.com/_/microsoft-dotnet-framework/)
* [Visual Studio Tools license](https://visualstudio.microsoft.com/license-terms/mlt031519/): applies to all [.NET Framework SDK container images](https://hub.docker.com/_/microsoft-dotnet-framework-sdk/)
```

Comment (mthalman) [reply eng/readme-templates/README.md]: Ok, made some updates.  Let me know how that looks.

File: README.aspnet.md [hunk 4]
```diff
@@ -116,6 +116,6 @@ See the [.NET Framework Lifecycle FAQ](https://support.microsoft.com/help/17455/
 
 # License
 
-* Legal Notice: [Container License Information](https://aka.ms/mcr/osslegalnotice)
-* The .NET Framework images use the same license as the [Windows Server Core base image](https://hub.docker.com/_/microsoft-windows-servercore/).
-* The [Visual Studio Tools license](https://visualstudio.microsoft.com/license-terms/mlt031519/) also applies to the [SDK images](https://hub.docker.com/_/microsoft-dotnet-framework-sdk/).
+* The [MICROSOFT SOFTWARE SUPPLEMENTAL LICENSE FOR WINDOWS CONTAINER BASE IMAGE](https://hub.docker.com/_/microsoft-windows-servercore/) applies to [].NET Framework container images](https://hub.docker.com/_/microsoft-dotnet-framework/).
```
Comment (richlander) [inline README.aspnet.md]: The one link is messed up.

Comment (richlander) [inline README.aspnet.md]: The one link is messed up.

Comment (mthalman) [reply README.aspnet.md]: That's from an old commit.  It's since been fixed.

File: eng/readme-templates/README.md (hunk 3 above)
Comment (richlander) [reply eng/readme-templates/README.md]: I like that. How about this small diff.

> * [Microsoft Container Images Legal Notice](https://aka.ms/mcr/osslegalnotice): applies to all [.NET Framework container images](https://hub.docker.com/_/microsoft-dotnet-framework/) 
> * [MICROSOFT SOFTWARE SUPPLEMENTAL LICENSE FOR WINDOWS CONTAINER BASE IMAGE](https://hub.docker.com/_/microsoft-windows-servercore/): applies to all [.NET Framework container images](https://hub.docker.com/_/microsoft-dotnet-framework/)
> * [Visual Studio Tools license](https://visualstudio.microsoft.com/license-terms/mlt031519/): applies to all [.NET Framework SDK container images](https://hub.docker.com/_/microsoft-dotnet-framework-sdk/)

**Review by MichaelSimons — APPROVED** [review]
LGTM

**Review by richlander — APPROVED** [review]
LGTM

**Pull request merged**